src/kvilib/ext/KviCryptEngine.cpp

```diff
@@ -157,7 +157,7 @@
 	KviCryptEngine::KviCryptEngine()
 	{
 #ifdef COMPILE_CRYPT_SUPPORT
-		setLastError(__tr2qs("Invalid crypt engine"));
+		setLastError(__tr2qs("Invalid encrypting engine"));
```

Comment:
It should probably be either `crypt` or `encryption` since `encrypting` would imply that it's a one-way system